### PR TITLE
Button

### PR DESCRIPTION
**Pull request by theprogrammerinyou**
Added Button to the component library. 
 - Primary Button - background-color :- red
 - Link Button - background-color :- accent color
 - Floating Action Button and Icon Button with same background color as a link button. 
The accent color and primary color are added according to the colors mentioned in the PRD document of the component library.

### CODE REVIEW

File: components/avatar/avatar.css [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    background-color: #bcc3ce;
+}
+
+.avatar-status.dnd {
+    background-color: #ff0000;
+}
+
+.avatar-status.online {
+    background-color: #32b643;
+}
```
Comment (nareshpoonia) [inline components/avatar/avatar.css]: For consistency, why don't you store these background colors in a variable as well, as you have done in rest of the code.

**Review by nareshpoonia — APPROVED** [review]
The code looks good, just one comment on the background-color, which might be stored in a variable for consistency.

File: components/avatar/avatar.css (hunk 1 above)
Comment (theprogrammerinyou) [reply components/avatar/avatar.css]: Hi, this was done in the respective branch.

File: components/alert/alert.css [hunk 2]
```diff
@@ -0,0 +1,29 @@
+@import "../../css/variables.css";
+
+.alert {
+    margin: 1rem;
+    padding: 1rem;
+    border: 1px solid white;
+    background-color: white;
+    color: black;
```
Comment (shuntzhere) [inline components/alert/alert.css]: Try not to use solid colors.

File: components/alert/alert.css [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    background-color: #00FF00;
+}
+
+.alert-primary {
+    background-color: #961ee1;
+}
+
+.alert-warning {
+    background-color: #d3e916;
+}
```
Comment (shuntzhere) [inline components/alert/alert.css]: Create variables for these as well and use them across.

File: components/alert/alert.html [hunk 4]
```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link href="alert.css" rel="stylesheet" />
+    <link href="../../css/base.css" rel="stylesheet" />
```
Comment (shuntzhere) [inline components/alert/alert.html:10]: You can import alert.css to base.css.

File: components/avatar/avatar.css [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+
+.avatar-status {
+    border: 2px solid #fff;
+    background-color: #bcc3ce;
+    padding: 8%;
+    border-radius: 50%;
+    position: absolute;
+    bottom: 0em;
+    right: 0em;
```
Comment (shuntzhere) [inline components/avatar/avatar.css]: I think you can just give 0 for this.

File: components/button/button.css [hunk 6]
```diff
@@ -0,0 +1,33 @@
+@import "../../css/variables.css";
+
+.button {
+    border-radius: 40px;
+    border: none;
+    color: black;
```
Comment (shuntzhere) [inline components/button/button.css]: Remove solid colors.

**Review by shuntzhere — APPROVED** [review]
Overall looks good, suggested some changes. Do give a look.

**Comment by theprogrammerinyou** [comment]
Done with suggestions. Two reviews done, closing this PR.

**Pull request closed without merging**